src/openrct2/ride/Ride.cpp

```diff
@@ -1590,6 +1602,24 @@ void RidePrepareBreakdown(Ride& ride, int32_t breakdownReason)
             }
             break;
     }
+#ifdef ENABLE_SCRIPTING
+    auto& hookEngine = GetContext()->GetScriptEngine().GetHookEngine();
+    if (hookEngine.HasSubscriptions(HookType::rideBreakDown))
+    {
+        auto ctx = GetContext()->GetScriptEngine().GetContext();
+        auto obj = DukObject(ctx);
+        obj.Set("ride", ride.id.ToUnderlying());
```

Comment:
```suggestion
        obj.Set("rideId", ride.id.ToUnderlying());
```

Otherwise it's not consistent with your `openrct2.d.ts` interface.